src/main/java/net/snowflake/client/core/QueryContextCache.java

```diff
@@ -37,11 +32,11 @@ public class QueryContextCache {
 
   private static final SFLogger logger = SFLoggerFactory.getLogger(QueryContextCache.class);
 
-  // Arrow format position
-  private static final int ID_POS = 0;
-  private static final int TIMESTAMP_POS = 1;
-  private static final int PRIORITY_POS = 2;
-  private static final int CONTEXT_POS = 3;
+  private static ObjectMapper mapper;
```

Comment:
nit: I would name this better befitting the use case.

Reply:
I rename it to `jsonObjectMapper`